lnbits/db.py

```diff
@@ -47,20 +49,13 @@ def _parse_timestamp(value, _):
             lambda value, curs: float(value) if value is not None else None,
         )
     )
-    register_type(
-        new_type(
-            (1082, 1083, 1266),
-            "DATE2INT",
-            lambda value, curs: time.mktime(value.timetuple())
-            if value is not None
-            else None,
-        )
-    )
 
     register_type(new_type((1184, 1114), "TIMESTAMP2INT", _parse_timestamp))
 else:
     if os.path.isdir(settings.lnbits_data_folder):
         DB_TYPE = SQLITE
+        sqlite3.register_converter("json", lambda val: json.loads(val.decode("utf-8")))
```

Comment:
- can `val` be `None`?

Reply:
I don't think so
https://docs.python.org/3/library/sqlite3.html#sqlite3.register_converter

> The converter is invoked for all SQLite values of type typename; it is passed a [bytes](https://docs.python.org/3/library/stdtypes.html#bytes) object and should return an object of the desired Python type

Reply:
- what happens if the value in the SQL column is `NULL` ?

Reply:
just tried it out, the converter is not called in that case and it just gives a `None`. should I add that in the test?